tools/GenerateDocFxStructure/README.md

```diff
@@ -0,0 +1,32 @@
+# DocFx file structure generator
+
+In order to reuse the README for each binding and reference them from the source comments (triple slash), the files need to be moved into a specific directory.
+
+This directory cannot include the images and other medias, they need to be moved in another folder. 
+
+The links to those media files need to be adjusted as well.
+
+On top, the links on the internal structure need to be adjusted as well from relative to absolute.
+
+## Usage
+
+The tool can be used like this:
+
+`GenerateDocFxStructure -d includesDirectory -m mediaDirectory -s sourceDirectory -r repoUrl`
+
+- `includesDirectory`: the includes destination directory. ex: `c:\iot-page-docfx\includes`
+- `mediaDirectory`: the media destination directory. ex: `c:\iot-page-docfx\media`
+- `sourceDirectory`: the source directory. ex: `c:\iot\src\devices`. It is important to point on the devices directory where all the markdown files will be extracted and analyzed.
```

Comment:
can you also update https://github.com/dotnet/iot/blob/main/Documentation/creating-new-release.md with an example invocation using some easy to adjust folder name?